src/integrationTest/java/uk/nhs/hee/tis/trainee/forms/api/LtftResourceIntegrationTest.java

```diff
@@ -95,59 +99,84 @@ void tearDown() {
     template.findAllAndRemove(new Query(), LtftForm.class);
   }
 
-  @Test
-  void shouldBeForbiddenFromGettingLtftFormWhenNoToken() throws Exception {
-    mockMvc.perform(get("/api/ltft/formX"))
-        .andExpect(status().isForbidden())
-        .andExpect(jsonPath("$").doesNotExist());
-  }
 
-  @Test
-  void shouldBeForbiddenFromGettingLtftFormWhenTokenLacksTraineeId() throws Exception {
-    String token = TestJwtUtil.generateToken("{}");
-    mockMvc.perform(get("/api/ltft/formX")
-            .header(HttpHeaders.AUTHORIZATION, token))
+  @ParameterizedTest
+  @CsvSource(delimiter = '|', textBlock = """
+      GET | /api/ltft/formX
+      """)
+  void shouldReturnForbiddenWhenNoToken(HttpMethod method, URI uri) throws Exception {
```

Comment:
This and similar tests have been refactored as parameterized tests to reduce boilerplate.